src/Client/Webhook.php

```diff
@@ -115,4 +188,31 @@ public function deleteWebhook(string $storeId, string $webhookId): void
             throw $this->getExceptionByStatusCode($method, $url, $response);
         }
     }
+
+    /**
+     * DEPRECATED: Please use `getStoreWebhooks()`.
```

Comment:
We should use phpDocumentor syntax @deprecated and @see here as it will show up in IDEs nicely as deprecated with warnings. https://docs.phpdoc.org/guide/references/phpdoc/tags/deprecated.html

e.g. 
```
@deprecated 2.0.0 Please use `getStoreWebhooks()` instead.
@see getStoreWebhooks()
```

Reply:
Thank you for teaching this syntax to me. Updated.